cmd/root.go

```diff
@@ -126,6 +126,11 @@ func init() {
 		"File containing password to your Konnect account")
 	viper.BindPFlag("konnect-password-file",
 		rootCmd.PersistentFlags().Lookup("konnect-password-file"))
+
+	rootCmd.PersistentFlags().String("konnect-addr", "https://konnect.konghq.com",
```

Comment:
Why do these flags need to be on `rootCmd` and not on `konnectCmd`?

Reply:
Because `konnect-email` and `konnect-password` are at root-level.
The question then comes up: why are these at root-level?
Because we want users to have a `~/.deck.yml` file that contains these credentials and not supply them on the CLI. So, because of viper working at the root-level only.